config/application.yml

```diff
@@ -41,3 +42,6 @@ hedera:
     consensus-message:
       topic_id:
       send_deadline: 300
+    scheduled-transaction:
```

Comment:
as stated in the other comment, it might be better to name the config as `burn-event`